src/System.Linq/tests/ExceptTests.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public class ExceptTests
+    {
+        // Class which is passed as an argument for EqualityComparer
+        private class AnagramEqualityComparer : IEqualityComparer<string>
```

Comment:
It is identical. While I've adapted it a bit from the original (equals has some short-circuits, quicker hash code that is likely less good distribution but that doesn't matter on the sizes used here) I think merging it is best left until the end, as otherwise it will just lead to the batches of transfers in each PR either depending on each other in a particular path, or else conflicting with each other.